src/controller/index.py

```diff
@@ -11,8 +12,14 @@
 templates.env = ExtendedEnvironment(loader=FileSystemLoader("src/assets/"))
 templates.env.filters['truncate'] = templates.env.truncate
 
+SECRET_KEY = os.environ.get("SECRET_KEY")
+ALGORITHM = os.environ.get("ALGORITHM")
```

Comment:
If this happens again (which it does ...) you have to create a file in the path /src/config/config.py with the reading of these variables

Reply:
Done